### ISSUE TITLE
Trust Dropbox security certificates

### ISSUE THREAD

**Issue by bstamps** [MGMIC/portal#3]
For outside partners who might use dropbox as a method of storing large file, the MGMIC portal should trust Dropbox security certificates.

**Comment by mbstacy** [comment]
@bstamps @bwawrik Yes this is very possible to interrogate url and determine if from dropbox. Then just need to change the end of the url from ?dl=0 to ?dl=1. Then the wget will work.
